### PR TITLE
W-11189469-log-forwarding-disable

### PR DESCRIPTION
**Pull request by sarathecoubian**
# Writer's Quality Checklist

Before merging your PR, did you:

- [ ] Run spell checker
- [ ] Run link checker to check for broken xrefs
- [ ] Check for orphan files
- [ ] Perform a local build and do a final visual check of your content, including checking for:
  - Broken images
  - Dead links
  - Correct rendering of partials if they are used in your content
  - Formatting issues, such as:
    - Misnumbered ordered lists (steps) or incorrectly nested unordered lists
    - Messed up tables
    - Proper indentation
    - Correct header levels
- [ ] Receive final review and signoff from:
  - Technical SME
  - Product Manager
  - Editor or peer reviewer
  - Reporter, if this content is in response to a reported issue (internal or external feedback)
- [ ] If applicable, verify that the software actually got released

### CODE REVIEW

File: modules/ROOT/pages/am-installing.adoc [hunk 1]
```diff
@@ -135,17 +135,29 @@ If you need to update the Anypoint Monitoring agent at some point in the future,
 `./bin/uninstall`
 . Delete the `am` folder.
 . Follow the instructions for <<install_ap_monitoring_onprem,installing the Anypoint Monitoring agent>> on-premises.
-. Start Mule.
+. Restart Mule.
 
 == Performance Impact
 
-Enabling Monitoring can impact CPU utilization and memory utilization, which varies depending upon specific application configuration. MuleSoft recommends that you test any performance impact in your pre-production environment prior to enabling monitoring in your production environment to ensure optimal performance and sizing.
+Enabling monitoring can impact CPU utilization and memory utilization, which varies depending upon specific application configuration. MuleSoft recommends that you test any performance impact in your pre-production environment prior to enabling monitoring in your production environment to ensure optimal performance and sizing.
```
Comment (IsaacEldridge) [inline modules/ROOT/pages/am-installing.adoc:142]:
```suggestion
Enabling monitoring can impact CPU utilization and memory utilization depending on your specific application configuration. Before enabling monitoring in your production environment, test any performance impact in your pre-production environment to ensure optimal performance and sizing.
```

File: modules/ROOT/pages/am-installing.adoc [hunk 2]
```diff
@@ -135,17 +135,29 @@ If you need to update the Anypoint Monitoring agent at some point in the future,
...
 
 == Troubleshooting
 
 - If you find connection failure or retry entries, ensure that port 443 in your outbound firewall is open.
 - If you do not find connection failure or retry entries, contact MuleSoft technical support.
 
+== Disable Log Forwarding for Hybrid Runtime Servers
+
+If you have a Titanium subscription and a hybrid runtime server, you can disable log forwarding to Anypoint Monitoring. You must have AM installer v.2.5.10 or later to disable this feature using a JVM property. You must configure this property manually on each hybrid server for which you want to disable log forwarding, and you cannot disable log forwarding per application.
```
Comment (IsaacEldridge) [inline modules/ROOT/pages/am-installing.adoc:151]:
```suggestion
If you have a Titanium subscription and a hybrid runtime server, you can disable log forwarding to Anypoint Monitoring if you have AM installer v.2.5.10 or later. To disable this feature, you must configure a JVM property on each hybrid server for which you want to disable log forwarding. You cannot disable log forwarding per application.
```

File: modules/ROOT/pages/am-installing.adoc [hunk 3]
```diff
@@ -135,17 +135,29 @@ If you need to update the Anypoint Monitoring agent at some point in the future,
...
+
+To disable log forwarding:
+
+. Upgrade the Anypoint Monitoring agent to a version that supports disabling log forwarding (am-2.5.10.0.zip or later).
+. Stop.
```
Comment (IsaacEldridge) [inline modules/ROOT/pages/am-installing.adoc:156]: LoL

Comment (sarathecoubian) [reply modules/ROOT/pages/am-installing.adoc:156]:
```suggestion
. Stop Mule.
```

Comment (sarathecoubian) [reply modules/ROOT/pages/am-installing.adoc:156]: 😂

**Review by IsaacEldridge — APPROVED** [review]

**Pull request merged**